packages/typography/package.json

```diff
@@ -19,7 +19,7 @@
     "start": "../../utils/scripts/start.sh"
   },
   "dependencies": {
-    "@zendeskgarden/css-variables": "6.3.0"
+    "@zendeskgarden/css-variables": "6.3.8"
```

Comment:
Shouldn't this be in `devDependencies`?

Reply:
These are imported in the typography components. So they have to be marked as a `dependency` since we mark everything as an `external` in Webpack.

I've also moved this to an open range (`^6.3.8`) as well.